node/src/commands/BitmapIndexType.ts

```diff
@@ -0,0 +1,17 @@
+/**
+ * Copyright Valkey GLIDE Project Contributors - SPDX Identifier: Apache-2.0
+ */
+
+/**
+ * Enumeration specifying if index arguments are BYTE indexes or BIT indexes.
+ * Can be specified in {@link BitOffsetOptions}, which is an optional argument to the {@link BaseClient.bitcount|bitcount} command.
```

Comment:
I think you need to copy imports to make links work.

Reply:
How can you check this? For example, even though I haven't added the import for doc symbols here, I can still cmd + click the BitOffsetOptions link here and it will take me to that type. I'm not sure if that means the docs will work or if that only works in my IDE. Additionally,  the {@link BaseClient.bitcount|bitcount} does not work as a clickable link for me even with the import unless I remove the "|bitcount" portion. Does that mean that {@link BaseClient.bitcount|bitcount} is not valid node doc syntax?

Reply:
Good question. According to [this doc](https://tsdoc.org/pages/tags/link/) and [that doc](https://github.com/microsoft/FluidFramework/wiki/TSDoc-Guidelines#link-aliasing-example), the link syntax is correct. Probably we have different link/doc renderers in our IDEs.

Reply:
We won't be able to verify this until https://github.com/valkey-io/valkey-glide/issues/556 is investigated and completed.  It'll depend on our javadoc generator.